themes/salix/layouts/partials/header.html

```diff
@@ -3,6 +3,9 @@
 </a>
 <header>
 	<h1>{{ replace (.Site.Title | plainify) " " "<br/>" | safeHTML}}.</h1>
+	<div aria-hidden="true">
```

Comment:
Thinking about this more, this will always be centered no? Why make it an element at all? Wasn't a background good enough? It seems more semantically correct.

Reply:
When initially laying this out I was concerned, but don't know the answer, about people who have a visual impairment but are not outright blind. People who might use a mixture of methods to look at content and lean on accessibility tools here and there when they need to.

Having said that, I can no longer get my phone to read me the alt-text for the image or see any other way to view it so reverting back to a background image makes sense.